crane_x7_tests/launch/pick_and_place.launch.py

```diff
@@ -0,0 +1,148 @@
+import os
+import sys
+from datetime import datetime
+from time import sleep
+
+import pytest
+import unittest
+import launch_testing
+from launch import LaunchDescription
+from launch.actions import ExecuteProcess, TimerAction
+from launch_ros.actions import Node
+
+from ament_index_python.packages import get_package_share_directory
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+
+from launch_ros.substitutions import FindPackageShare
+from launch.substitutions import PathJoinSubstitution
+import launch_testing.markers
+
+import subprocess
+import re
+
+
+@pytest.mark.launch_test
+@launch_testing.markers.keep_alive
+
+def generate_test_description():
+    proc_env = os.environ.copy()
+    proc_env["PYTHONUNBUFFERED"] = "1"
+
+    sim = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("crane_x7_gazebo"),
+                        "launch",
+                        "crane_x7_with_table.launch.py",
+                    ]
+                )
+            ]
+        ),
+    )
+
+    demo = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("crane_x7_examples"),
+                        "launch",
+                        "example.launch.py",
+                    ]
+                )
+            ]
+        ),
+        launch_arguments={
+            "use_sim_time": "true", 
+            "example": "pick_and_place"
+        }.items(),
+    )
+
+    
+    return LaunchDescription(
+        [
+            sim,
+            TimerAction(period=15.0, actions=[demo]),  # Need to optimize period, if under 10 seconds will not run
```

Comment:
Thank you, nice to have a comment for those.

Indeed, we may want a better solution at some point, but I think this is good as is for now